### DISCUSSION TITLE
Array index value

### DISCUSSION THREAD

**Discussion by rohankr261** [rohankr261/dsa_rohan#2] [Q&A] [answered]
Can anyone tell me what if we dont assign any array with any specific number then what will be it gonna return ?

**Comment by Abhineshhh** [comment, marked as answer]
> Can anyone tell me what if we dont assifn any array with any specific number then what will be it gonna return ?

it will depend on code .!
when u use
int arr[4];
cout<<arr[2];
its output will be garbage value

but
when the condition is
int arr[5]{};
cout<<arr[2];
it will be zero because all the elements of array is assigned with zero .!

**Reply by rohankr261** [reply]
okk thanksssss
